editor/filesystem_dock.h

```diff
@@ -203,6 +203,7 @@ class FileSystemDock : public VBoxContainer {
 	CreateDialog *new_resource_dialog = nullptr;
 
 	bool always_show_folders = false;
+	uint8_t thumbnail_size_setting = 0;
```

Comment:
I believe we wouldn't micro-optimize this kind of thing, the return value of the project setting is `int` and this will later be promoted to `int` anyway, so let's keep it `int`.